<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>457</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Hotel Booking</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>531</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="1" column="0" colspan="4">
      <widget class="QTableView" name="hotelView">
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Current time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Rooms:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <widget class="QMenu" name="menuImport_CSV">
     <property name="title">
      <string>Import CSV</string>
     </property>
     <addaction name="actionImportRooms"/>
     <addaction name="actionImportUsers"/>
    </widget>
    <widget class="QMenu" name="menuExport_CSV">
     <property name="title">
      <string>Export CSV</string>
     </property>
     <addaction name="actionExportRooms"/>
     <addaction name="actionExportUsers"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSaveUsers"/>
    <addaction name="actionOpenUsers"/>
    <addaction name="separator"/>
    <addaction name="menuImport_CSV"/>
    <addaction name="menuExport_CSV"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddRoom"/>
    <addaction name="actionEditRoom"/>
    <addaction name="actionListUsers"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRegister"/>
    <addaction name="actionAuth"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open file with rooms</string>
   </property>
  </action>
  <action name="actionAddRoom">
   <property name="text">
    <string>Add Room</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save file with rooms</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
  <action name="actionEditRoom">
   <property name="text">
    <string>Edit Room</string>
   </property>
  </action>
  <action name="actionListUsers">
   <property name="text">
    <string>List Users</string>
   </property>
  </action>
  <action name="actionSaveUsers">
   <property name="text">
    <string>Save file with users</string>
   </property>
  </action>
  <action name="actionOpenUsers">
   <property name="text">
    <string>Open file with users</string>
   </property>
  </action>
  <action name="actionAuth">
   <property name="text">
    <string>Auth</string>
   </property>
  </action>
  <action name="actionImportRooms">
   <property name="text">
    <string>Rooms</string>
   </property>
  </action>
  <action name="actionImportUsers">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
  <action name="actionExportRooms">
   <property name="text">
    <string>Rooms</string>
   </property>
  </action>
  <action name="actionExportUsers">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openRoomsTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddRoom</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>create_room()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveRoomsTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRegister</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>registerUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hotelView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>booking_room(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditRoom</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>edit_room()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionListUsers</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>list_user()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveUsers</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveUsers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenUsers</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openUsers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAuth</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>authUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>windowAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImportRooms</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ImportRoomsCSV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportRooms</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ExportRoomsCSV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImportUsers</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ImportUsersCSV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportUsers</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ExportUsersCSV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exit()</slot>
  <slot>openRoomsTable()</slot>
  <slot>create_room()</slot>
  <slot>saveRoomsTable()</slot>
  <slot>registerUser()</slot>
  <slot>edit_room()</slot>
  <slot>booking_room(QModelIndex)</slot>
  <slot>list_user()</slot>
  <slot>saveUsers()</slot>
  <slot>openUsers()</slot>
  <slot>authUser()</slot>
  <slot>windowAbout()</slot>
  <slot>ImportRoomsCSV()</slot>
  <slot>ExportRoomsCSV()</slot>
  <slot>ImportUsersCSV()</slot>
  <slot>ExportUsersCSV()</slot>
 </slots>
</ui>
